src/lib.rs

```diff
@@ -434,39 +435,6 @@ let mut node = RawNode::new(&mut config, store, &logger).unwrap();
 node.raft.become_candidate();
 node.raft.become_leader();
 
-// Call this on the leader, or send the command via a normal `MsgPropose`.
-node.raft.propose_membership_change((
```

Comment:
Why remove the example?

Reply:
joint consensus is removed in this PR.

Reply:
But it still needs an example to do conf change, right?

Reply:
How about add it after the new API is introduced? It will be
```rust
raw_node.propose_conf_change(cc: &ConfChangeV2)
```
We don't use `ConfChangeV2` for now.